lib/src/arithmetic/floating_point/floating_point_converter.dart

```diff
@@ -200,34 +212,45 @@ class FloatingPointConverter<FpTypeIn extends FloatingPoint,
           mux(leadOneValid, leadOnePre.zeroExtend(maxExpWidth), biasDiff)
               .named('leadOne');
 
-      final seW = se.zeroExtend(maxExpWidth);
-      final newSe = mux(
-          leadOneValid & Const(source.explicitJBit) & source.isNormal,
-          mux(seW.gte(leadOne), seW - (leadOne - Const(1, width: maxExpWidth)),
-              Const(0, width: maxExpWidth)),
-          seW);
+      final seW = se.zeroExtend(maxExpWidth).named('sourceExpExt');
+      final computedExpFull = mux(
+              leadOneValid & Const(source.explicitJBit) & source.isNormal,
+              mux(
+                      seW.gte(leadOne),
+                      seW - (leadOne - Const(1, width: maxExpWidth)),
+                      Const(0, width: maxExpWidth))
+                  .named('computedExp'),
+              seW)
+          .named('computedExpFull');
 
       final nextShift = mux(
-          biasDiff.gte(newSe),
-          (source.isNormal.zeroExtend(maxExpWidth).named('srcIsNormal') +
-                  (biasDiff - newSe).named('negSourceRebiased'))
-              .named('shiftSubnormal'),
-          Const(0, width: maxExpWidth));
-
-      final tns = nextShift -
-          (se - newSe) +
-          mux(Const(source.explicitJBit), Const(-1, width: maxExpWidth),
-              Const(0, width: maxExpWidth)) +
-          mux(Const(destination.explicitJBit), Const(1, width: maxExpWidth),
-              Const(0, width: maxExpWidth));
+              biasDiff.gte(computedExpFull),
+              (source.isNormal.zeroExtend(maxExpWidth).named('srcIsNormal') +
```

Comment:
this is really a lot of `.named`s everywhere.  Is it worth it?

Reply:
I was thinking of automating the check for clean SystemVerilog.  So this is zero-tolerance.  We may not want it, but these names quickly expand.  If you dont rename the zeroExtend, you get a swizzle. Then if  you add it to something, you can quickly get to swizzle_add__swizzle__add__subset....  It is great that the base system does this, don't get me wrong.  But if the metric is that this naming is in the components, then that means that people who write clean ROHD but use components get these names in their Verilog.  While we are trying to convince people about debuggability, I thought it would be good to get to zero strange names.

The current PR has *no* (as far as I have tested) names with '__'. (If you combine it with the definitionName PR).

Reply:
I think there are better ways to write in ROHD that don't end up with so many nameds in one spot.  Like splitting them out of complex logic or assigning to Logics that are named already.

One I ran across was
```dart
count += (complex logic);
```
This resulted in a super long name __count__count__count__count ....  something like that.

It was FAR cleaner Verilog to write it as:
```dart
count = count + (complex Logic).named('increment');
```
And not using the += operator was actually beneficial to reading the code.

Reply:
Oh another thing I discovered.  There is probably a much better way to disposition Bool parameters by NOT converting them to Logic, but instead using var-style code.  This makes things far cleaner.

var  myLogic = <complex logic>;

if (parameter) {
      myLogic = <modified logic, including myLogic>
}

All this jbit code can come out more clean this way.   But for now, I was trying to get clean Verilog first.